### DISCUSSION TITLE
Looking for subclipse 1.6.18 plugin for old version of eclipse

### DISCUSSION THREAD

**Discussion by eric-huynh** [subclipse/subclipse#285] [Q&A]
Hi All,
I'm looking for version 1.6.18 of subclipse to install the plugin on an old eclipse/rad we use for an old application.
Has anyone of you still got this version file so I can update my eclipse through a local site, please ?
Thanks in advance for your help.
E.

**Comment by markphip** [comment]
You can download the zipped archives for any previous release here: https://github.com/subclipse/updates/tree/main/subclipse

**Reply by eric-huynh** [reply]
Thanks for pointing me to the right direction !
Keep up the good work !
